### PR TITLE
[SeaTunnel #998] add an environment variable to control print AsciiLogo

### LINKED ISSUE #998: [Feature][AsciiLogo] add an environment variable to control print AsciiLogo
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

add an environment variable to control print AsciiLogo, whether to show any string or hide.

### Usage Scenario

User can custom ascii logo, even they can hide the AsciiLogo when they don't like it.

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by yx91490**
## Purpose of this pull request

close #998 

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by yx91490** [comment]
may be blocked by https://github.com/apache/incubator-seatunnel/issues/999

File: config/seatunnel-env.sh [hunk 1]
```diff
@@ -19,4 +19,7 @@
 # Home directory of spark distribution.
 SPARK_HOME=${SPARK_HOME:-/opt/spark}
 # Home directory of flink distribution.
-FLINK_HOME=${FLINK_HOME:-/opt/flink}
\ No newline at end of file
+FLINK_HOME=${FLINK_HOME:-/opt/flink}
+
+# Control the ascii logo string, set to empty to disable it
+export SEATUNNEL_ASCII_LOGO=SeaTunnel
```
Comment (leo65535) [inline config/seatunnel-env.sh:25]: hi @yx91490, it seems not necessary let `SEATUNNEL_ASCII_LOGO` as the system env.

Comment (yx91490) [reply config/seatunnel-env.sh:25]: what's your suggestion?

Comment (leo65535) [reply config/seatunnel-env.sh:25]: We may never change the logo, so it should be a final constant.

Comment (wuchunfu) [reply config/seatunnel-env.sh:25]: > hi @yx91490, it seems not necessary let `SEATUNNEL_ASCII_LOGO` as the system env.

+1

Comment (yx91490) [reply config/seatunnel-env.sh:25]: The main purpose is hide the logo print, change the logo word is just a fun for some body who want to custom.

Comment (cody1993) [reply config/seatunnel-env.sh:25]: > hi @yx91490, it seems not necessary let `SEATUNNEL_ASCII_LOGO` as the system env.

+1

Comment (zhongjiajie) [reply config/seatunnel-env.sh:25]: I agree with @leo65535 , hard code with the logo is better. But I do not have strong opinions here.

**Comment by garyelephant** [comment]
LGTM

**Review by garyelephant — APPROVED** [review]

**Pull request merged**